ui/src/app/workflows/components/workflow-details/workflow-details.tsx

```diff
@@ -129,6 +129,25 @@ export const WorkflowDetails = ({history, location, match}: RouteComponentProps<
                     });
                 });
         }
+
+        if (workflow?.spec?.workflowTemplateRef) {
+            const templateName: string = workflow.spec.workflowTemplateRef.name;
+            const clusterScope: boolean = workflow.spec.workflowTemplateRef.clusterScope;
+            const url: string = clusterScope ? `/cluster-workflow-templates/${templateName}` : `/workflow-templates/${workflow.metadata.namespace}/${templateName}`;
+
+            const templateLink: Link = {
+                name: 'Workflow Template Link',
```

Comment:
Please change to “Open Workflow Template” and use the same icon as left-hand-nav does.